### PR TITLE
Create Resources

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2244](https://jira.bigcommerce.com/browse/DEVDOCS-2244)

## What changed?
Created a new article called Resources. 
This article contains a small blurb and links to additional documentation for the following items:
- API Clients
- App Starters
- BigDesign
- SDKs

### CODE REVIEW

File: docs/api-docs/partner/Resources [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+### On this page
+- [API clients](#API-clients)
+- [App starters](#App-starters)
+- [BigDesign](#BigDesign)
+- [SDK (Checkout SDK)](#SDK)
+</div> 
+
+## API clients
+Accelerate your app development with BigCommerce API clients. The API clients are available in different programming languages such as Python, Php, Ruby, and JavaScript. See [Tools & Resources](https://developer.bigcommerce.com/tools-resources) for a list of our API clients.
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:13]:
```suggestion
Accelerate your app development with BigCommerce API clients. BigCommerce API clients are available for the following languages:

- [Python](https://github.com/bigcommerce/bigcommerce-api-python)
- [PHP](https://github.com/bigcommerce/bigcommerce-api-php)
- [Ruby](https://github.com/bigcommerce/bigcommerce-api-ruby)
- [JavaScript](https://github.com/getconversio/node-bigcommerce)
```

File: docs/api-docs/partner/Resources [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## App starters
+There are apps that every BigCommerce Enterprise customer should have to enhance their customer experience and sell more online. See [The BigCommerce Enterprise App Starter Kit](https://www.groovecommerce.com/ecommerce-blog/bigcommerce-enterprise-app-starter-kit/) for a list of the must-have apps for all store owners.
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:16]: I am hesitant to link out to external resources. Confirmed with Corey that we can link to sample apps (see Hello World Apps in [Tools](https://developer.bigcommerce.com/tools-resources)). 
```suggestion
BigCommerce sample apps are open-source projects developed by the BigCommerce team and community. For inspiration and to see the code behind each project, visit each individual repository.

- [Python App](https://github.com/bigcommerce/hello-world-app-python-flask)
- [PHP App](https://github.com/bigcommerce/hello-world-app-php-silex)
- [Ruby App](https://github.com/bigcommerce/hello-world-app-ruby-sinatra)
- [OmniAuth Bigcommerce Strategy](https://github.com/bigcommerce/omniauth-bigcommerce)
-[AWS Amplify App](https://github.com/hatertron3000/amplify-bigcommerce)

```

File: docs/api-docs/partner/Resources [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- [API clients](#API-clients)
+- [App starters](#App-starters)
+- [BigDesign](#BigDesign)
+- [SDK (Checkout SDK)](#SDK)
+</div> 
+
+## API clients
+Accelerate your app development with BigCommerce API clients. The API clients are available in different programming languages such as Python, Php, Ruby, and JavaScript. See [Tools & Resources](https://developer.bigcommerce.com/tools-resources) for a list of our API clients.
+
+## App starters
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:15]:
```suggestion
## Sample apps
```

File: docs/api-docs/partner/Resources [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+There are apps that every BigCommerce Enterprise customer should have to enhance their customer experience and sell more online. See [The BigCommerce Enterprise App Starter Kit](https://www.groovecommerce.com/ecommerce-blog/bigcommerce-enterprise-app-starter-kit/) for a list of the must-have apps for all store owners. 
+
+## BigDesign
+BigCommerce developed a collection of reusable React components, design guidelines, and UI patterns. This collection, known collectively as [BigDesign](https://developer.bigcommerce.com/api-docs/apps/guide/ui), allows you to rapidly develop an app's front-end that is consistent with BigCommerce's UI.
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:19]:
```suggestion
[BigDesign](https://developer.bigcommerce.com/api-docs/apps/guide/ui) is a collection of reusable React components, design guidelines, and UI patterns that allows you to rapidly develop an app's front-end that is consistent with BigCommerce's UI. 
```

File: docs/api-docs/partner/Resources [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+## SDK (Checkout SDK)
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:21]:
```suggestion
## Checkout SDK
```

File: docs/api-docs/partner/Resources [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Resources
+
+<div class="otp" id="no-index">
+
+### On this page
+- [API clients](#API-clients)
+- [App starters](#App-starters)
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:7]:
```suggestion
- [Sample apps](#sample-apps)
```

File: docs/api-docs/partner/Resources [hunk 7]
```diff
@@ -0,0 +1,22 @@
+# Resources
+
+<div class="otp" id="no-index">
+
+### On this page
+- [API clients](#API-clients)
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:6]:
```suggestion
- [API clients](#api-clients)
```

File: docs/api-docs/partner/Resources [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+- [App starters](#App-starters)
+- [BigDesign](#BigDesign)
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:8]:
```suggestion
- [BigDesign](#bigdesign)
```

File: docs/api-docs/partner/Resources [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+- [SDK (Checkout SDK)](#SDK)
```
Comment (ektravel) [inline docs/api-docs/partner/Resources:9]:
```suggestion
- [Checkout SDK](#checkout-sdk)
```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**